snuba/cli/consumer.py

```diff
@@ -139,21 +140,33 @@ def consumer(
     setup_logging(log_level)
     setup_sentry()
     logger.info("Consumer Starting")
-    storage_key = StorageKey(storage_name)
+    storage_keys = [
+        getattr(StorageKey, storage_name.upper()) for storage_name in storage_names
+    ]
 
-    metrics = MetricsWrapper(
-        environment.metrics,
-        "consumer",
-        tags={"group": consumer_group, "storage": storage_key.value},
-    )
-    configure_metrics(StreamMetricsAdapter(metrics))
+    if len(storage_keys) == 1:
+        metrics = MetricsWrapper(
+            environment.metrics,
+            "consumer",
+            tags={"group": consumer_group, "storage": storage_keys[0].value},
+        )
+        configure_metrics(StreamMetricsAdapter(metrics))
+    else:
+        metrics = MetricsWrapper(
+            environment.metrics,
+            "consumer",
+            tags={
+                "group": consumer_group,
+                "storage": "_".join([storage_keys[0].value, "m"]),
```

Comment:
What is this "m" about?

Reply:
I'm not completely sure, I got it from: https://github.com/getsentry/snuba/blob/master/snuba/cli/multistorage_consumer.py#L235-L242.

Reply:
We have alerts and consumer dashboards that are based on the storage tag. That's probably a bad idea, using the consumer group could be better.
That breaks with the multistorage consumer as there are multiple storages. That `m` was to tell apart the multistorage ones in our metrics.